playbook/app/pb_kits/playbook/pb_background/_background.jsx

```diff
@@ -6,19 +6,22 @@ import { buildAriaProps, buildCss, buildDataProps } from '../utilities/props'
 import { globalProps } from '../utilities/globalProps.js'
 
 type BackgroundProps = {
+  alt?: string,
   aria?: object,
   backgroundColor?: 'gradient' | 'dark' | 'light' | 'white' | 'success' | 'warning' | 'error' | 'info' | 'neutral' | 'primary' | 'category_1' | 'category_2' | 'category_3' | 'category_4' | 'category_5' | 'category_6' | 'category_7' | 'category_8' | 'category_9' | 'category_10' | 'category_11' | 'category_12' | 'category_13' | 'category_14' | 'category_15' | 'category_16' | 'category_17' | 'category_18' | 'category_19' | 'category_20' | 'category_21',
   children?: array<React.ReactNode> | React.ReactNode,
   className?: string,
   data?: object,
   id?: string,
-  imageUrl?: string,
+  imageUrl: string,
```

Comment:
Why `?` was removed?

Reply:
Good catch, I can't recall exactly why i removed it. But from what i do remember, i did move around a lot of parts in this file when we were testing to make sure that the image was rendering for the image prop. I think i just forgot to put it back in at some point. I just tested the transition prop to make sure it still works WITH the `?` added in for `imageUrl`. So i'm going to re-add it in case this causes a breaking change.  Thank you for your feedback, resolving this conversation